go/pkg/bertyprotocol/store_metadata.go

```diff
@@ -565,7 +565,7 @@ func (m *metadataStore) ContactRequestIncomingReceived(ctx context.Context, cont
 		return nil, errcode.ErrGroupInvalidType
 	}
 
-	if err := contact.CheckFormat(); err != nil {
+	if err := contact.CheckFormat(bertytypes.ShareableContactOptionsAllowMissingRDVSeed); err != nil {
```

Comment:
Am I missing something?

How before and after can both compile?

Reply:
Ok: `go/pkg/bertytypes/contact.go:func (m *ShareableContact) CheckFormat(options ...ShareableContactOptions) error {`